packages/web-components/src/option/README.md

```diff
@@ -0,0 +1,139 @@
+| `selectedAttributeChanged` | protected |                                                                     |                                           | `void` |                |
+| `selectedChanged`          | protected |                                                                     |                                           | `void` |                |
+| `initialValueChanged`      | public    |                                                                     | `previous: string, next: string`          | `void` |                |
+
+<br />
+
+### **Attributes**
+
+<br />
+
+| Name       | Field             |
+| ---------- | ----------------- |
+|            | disabled          |
+| `selected` | selectedAttribute |
+| `value`    | initialValue      |
+
+<br />
+
+### **CSS Parts**
+
+<br />
+
+| Name      | Description                      |
+| --------- | -------------------------------- |
+| `content` | Wraps the listbox option content |
+
+<br />
+
+### **Slots**
+
+<br />
+
+| Name    | Description                                                      |
+| ------- | ---------------------------------------------------------------- |
+|         | The default slot for listbox option content.                     |
+| `start` | Content which can be provided after the listbox option content.  |
+| `end`   | Content which can be provided before the listbox option content. |
+
+<br />
+<hr />
+<br />
+
+### **Suggested Template**
+
+<br />
+
+`listboxOptionTemplate()` from `FASTFoundation`
+
+<br />
+<hr />
+<br />
+
+## **Accessibility**
+
+[W3 Option Spec](https://w3c.github.io/aria/#option)
+
+<br />
+
+### **WAI-ARIA Roles, Methods, and IDL Properties**
+
+<br/>
+
+### Class: `DelegatesARIAListboxOption`
+
+<br/>
+
+**Fields**
+
+| Name           | Reflected ARIA Content Attribute | Privacy | Type                                  | Description                                                          | Default |
+| -------------- | -------------------------------- | ------- | ------------------------------------- | -------------------------------------------------------------------- | ------- |
+| `ariaChecked`  | `aria-checked`                   | public  | `"true" or "false" or string or null` | See https://www.w3.org/TR/wai-aria-1.2/#option for more information. |         |
+| `ariaPosInSet` | `aria-posinset`                  | public  | `string or null`                      | See https://www.w3.org/TR/wai-aria-1.2/#option for more information. |         |
+| `ariaSelected` | `aria-selected`                  | public  | `"true" or "false" or string or null` | See https://www.w3.org/TR/wai-aria-1.2/#option for more information. |         |
+| `ariaSetSize`  | `aria-setsize`                   | public  | `string or null`                      | See https://www.w3.org/TR/wai-aria-1.2/#option for more information. |         |
```

Comment:
This should be inherited and is essentially "internals" I don't think it needs to be enumerated...the presence of this is likely going to cause confusion. These lines illustrate that the exported `ListboxOption` includes this class.
<img width="701" alt="image" src="https://user-images.githubusercontent.com/13071055/231293447-12a25c3f-cf3b-44c0-8527-07e7309332c0.png">